src/main/resources/org/jpeek/metrics/OCC.xsl

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
```

Comment:
@llorllale Again, I don't understand why this PR adds both OCC metric and implements the metric.xsl thing?

Reply:
@amihaiemil I fixed this